xc7/tests/ps7/counter_emio/CMakeLists.txt

```diff
@@ -12,6 +12,5 @@ add_fpga_target(
 add_vivado_target(
     NAME counter_emio_zybo_vivado
     PARENT_NAME counter_emio_zybo
-    # Disabled until PS7 is supported in fasm2bels
-    DISABLE_DIFF_TEST
+    BIT_TO_V_EXTRA_ARGS --allow-non-dedicated-clk-routes
```

Comment:
I believe this is not required?

Reply:
Removing that will make Vivado complain about non-dedicated clock routes. From what I see the VPR tends to choose non-dedicated clock route between the clock input pin and a BUFG.

Reply:
@mkurc-ant - Can we log a tracking bug to investigate the use of non-dedicated clock routes?

Reply:
@mithro Ok, once I encounter that behavior again I'll file an issue with a concrete example. Meanwhile I removed the `--allow-non-dedicated-clk-routes` option.